### PR TITLE
 build - add 'deps' step that records output of 'npm ls' into build

### PR DESCRIPTION
**Pull request by kumavis**
for better historical dep checking

having hashes in there would be nice but this will do for now

### CODE REVIEW

**Comment by kumavis** [comment]
deps.txt is about 160K unzipped

**Comment by kumavis** [comment]
ok to merge : )
deploy when infura is ready

File: package.json [hunk 1]
```diff
@@ -93,8 +93,6 @@
     "extensionizer": "^1.0.0",
     "fast-json-patch": "^2.0.4",
     "fast-levenshtein": "^2.0.6",
-    "gulp": "github:gulpjs/gulp#4.0",
```
Comment (danfinlay) [inline package.json]: We don't need gulp anymore? Isn't this useful/important, esp. since it's a specific version?

**Review by danfinlay — COMMENTED** [review]
Small Q

File: package.json (hunk 1 above)
Comment (kumavis) [reply package.json]: sorry should have clarified!
 gulp was present in both dev-dep and regular deps. this removes it from regular deps.

**Review by danfinlay — APPROVED** [review]
LGTM

**Pull request merged**